be/src/storage/task/engine_publish_version_task.cpp

```diff
@@ -37,7 +37,7 @@ EnginePublishVersionTask::EnginePublishVersionTask(TTransactionId transaction_id
           _tablet_info(tablet_info),
           _rowset(rowset) {}
 
-Status EnginePublishVersionTask::finish() {
```

Comment:
before this change, finish is executed with all tablet lock held. after this change, lock is not held. Are you sure is it OK to do this change?

Reply:
`before this change, finish is executed with all tablet lock held`
No, which part of the code are you talking about?

Reply:
get_related_tablets returns empty, so it doesn't change anything